hs_core/templates/pages/genericresource.html

```diff
@@ -830,14 +830,13 @@ <h4 class="modal-title">Manage access</h4>
 
         {#        ======= Content =======#}
 
+        {% if cm.resource_type != "ToolResource" %}
```

Comment:
@mauriel133 Got it. By any chance, could we use this show_content_files flag to hide content section to this type of resource? If not, I am OK with your code change. Copy @ironfroggy, @pkdash, and @selimnairb for their reference as well.